test_schemas/cy/test_language.json

```diff
@@ -99,6 +96,104 @@
                                 ]
                             }
                         },
+                        {
+                            "type": "Question",
+                            "id": "number-of-people-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "number-of-people-answer",
+                                        "label": "Nifer y preswylwyr cartrefi",
```

Comment:
Most of the translations look OK except this one,  can it be changed to: "Nifer o breswylwyr yn y cartref"?